packages/@ama-sdk/schematics/README.md

```diff
@@ -21,18 +23,17 @@ npx -p @angular/cli ng add @ama-sdk/core
 else
 
 ```shell
-# Install @angular-devkit/schematics-cli, @ama-sdk/core and the SDK generator
-yarn add --dev @angular-devkit/schematics-cli @ama-sdk/schematics @ama-sdk/core
-yarn schematics @ama-sdk/schematics:install
-yarn schematics @ama-sdk/core:install
+npm create @ama-sdk typescript <project-name> -- [--spec-path=./path/to/spec.yaml]
```

Comment:
In fact the `(npm|yarn) create` is developed to create a repo from scratch not inside an otter project 
maybe it could have a dedicated section